script.js

```diff
@@ -21,4 +21,187 @@ function closeScreen() {
+    githubIcon: './images/github-log.svg',
+    liveIcon: './images/live-icon.svg',
+    ProjectLink: 'https://tayyabkm.github.io/Peronsal-Portfolio/',
+    GitHubLink: 'https://github.com/TayyabKM/Peronsal-Portfolio',
+  },
+  {
+    classNO: 'n4',
+    Subtitle: 'Keeping track of \nhundreds of components',
+    Technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    Description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    githubIcon: './images/github-log.svg',
+    liveIcon: './images/live-icon.svg',
+    ProjectLink: 'https://tayyabkm.github.io/Peronsal-Portfolio/',
+    GitHubLink: 'https://github.com/TayyabKM/Peronsal-Portfolio',
+  },
+  {
+    classNO: 'n5',
+    Subtitle: 'Keeping track of \nhundreds of components',
+    Technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    Description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    githubIcon: './images/github-log.svg',
+    liveIcon: './images/live-icon.svg',
+    ProjectLink: 'https://tayyabkm.github.io/Peronsal-Portfolio/',
+    GitHubLink: 'https://github.com/TayyabKM/Peronsal-Portfolio',
+  },
+  {
+    classNO: 'n6',
+    Subtitle: 'Keeping track of \nhundreds of components',
+    Technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    Description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    githubIcon: './images/github-log.svg',
+    liveIcon: './images/live-icon.svg',
+    ProjectLink: 'https://tayyabkm.github.io/Peronsal-Portfolio/',
+    GitHubLink: 'https://github.com/TayyabKM/Peronsal-Portfolio',
+  },
+];
```

Comment:
- [ ] Kindly make sure to add the picture (screenshot) of the app which will be displayed at the top of the pop container on each project. I have noticed that your pop doesn't have any project screenshot kindly make sure to add pictures to all project object in order to meet the design :wink: